Prace_domowe/Praca_domowa1/Grupa4/Hubert_Drążkowski/Hubert_Drazkowski_PD2.ipynb

```diff
@@ -0,0 +1,2196 @@
+{
```

Comment:
<p>Ogólnie to ok, ale pls formatuj tekst, bo czytanie takie ciągu jest bardzo niewygodne.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>